service/orchestrator/metrics.json

```diff
@@ -1,128 +1,450 @@
+      }
+    }
   },
   {
-    "id": "CustomerKeyEncryption"
+    "id": "CustomerKeyEncryption",
+    "name": "Storage Encryption: Customer Key Encryption",
+    "description": "This metric is used to assess if encryption is done via customer keys",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "IdentityPasswordPolicy"
+    "id": "IdentityPasswordPolicy",
+    "name": "Identity: Password Policy",
+    "description": "This metric is used to assess if a password policy is in place",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "IdentityRecentActivity"
+    "id": "IdentityRecentActivity",
+    "name": "Identity: Recent Activity",
+    "description": "This metric is used to assess if a user is still active (days)",
+    "category": "",
+    "scale": 3,
+    "range": {
+      "minMax": {
+        "min": 0,
+        "max": 365
+      }
+    }
   },
     {
-    "id": "L3FirewallEnabled"
+    "id": "L3FirewallEnabled",
+    "name": "L3 Firewall: Enabled",
+    "description": "This metric is used to assess if a L3 firewall is in place",
+    "category": "",
+    "scale": 1,
+      "range": {
+        "allowedValues": {
+          "values": [
+            true,
+            false
+          ]
+        }
+      }
   },
   {
-    "id": "L3FirewallRestrictedPorts"
+    "id": "L3FirewallRestrictedPorts",
+    "name": "L3 Firewall: Restricted Ports",
+    "description": "This metric is used to assess the restricted ports of a L3 firewall",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "*"
+        ]
+      }
+    }
   },
   {
-    "id": "MalwareProtectionEnabled"
+    "id": "MalwareProtectionEnabled",
+    "name": "Malware Protection: Enabled",
+    "description": "This metric is used to assess if Malware Protection is enabled",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "MalwareProtectionOutput"
+    "id": "MalwareProtectionOutput",
+    "name": "Malware Protection: Output",
+    "description": "This metric is used to assess to which service the Malware Protection results are sent",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "*"
+        ]
+      }
+    }
   },
   {
-    "id": "MutualAuthentication"
+    "id": "MutualAuthentication",
+    "name": "Mutual Authentication",
+    "description": "This metric is used to assess if mutual authentication, including client certificate, has been enabled",
+    "category": "",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "*"
```

Comment:
Shouldn't that be a bool?

Reply:
Yes, then it is more consistent. Bit it is almost a good example for a metric where our current notion of `scale` and `range` reaches its limits. Because it an more "complex" metric where we check multiple things, it is not so easy applicable. But in these case both checks are of boolean nature, so it is fine to change it.